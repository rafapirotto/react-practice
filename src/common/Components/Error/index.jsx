import React from 'react';

import error from '../../assets/error.png';

const Error = () => (
  <div className="mt-20 flex justify-center">
    <img src={error} alt="error" />
  </div>
);

export default Error;
