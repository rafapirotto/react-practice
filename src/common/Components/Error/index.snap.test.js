import React from 'react';
import renderer from 'react-test-renderer';

import Error from '.';

test('It matches the snapshot', () => {
  const component = renderer.create(<Error />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
