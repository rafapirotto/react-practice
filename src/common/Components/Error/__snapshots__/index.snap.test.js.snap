// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches the snapshot 1`] = `
<div
  className="mt-20 flex justify-center"
>
  <img
    alt="error"
    src="error.png"
  />
</div>
`;
